end-to-end-tests/cypress/support/commands.js

```diff
@@ -47,7 +47,9 @@ Cypress.Commands.add("navToEditorByDrawer", (oscalType, pageTitle) => {
   cy.get('ul[aria-label="file system navigator"] li', { timeout: 30000 })
     .should("have.attr", "aria-expanded", "false")
     .contains(oscalType)
-    .click();
+    .trigger("mouseover")
+    .wait(10000)
+    .click({ force: true });
```

Comment:
This still doesn't work but is where the issue lies I think.

For only `Catalogs`, the click fails. You can manually click it and things work. You can have `.click().click()` and the catalog tests pass (everything else fails though).

I am unsure why it is clicking when the box is not yet clickable....